templates/new-layout.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons/font/bootstrap-icons.css" rel="stylesheet">
+    <link rel="stylesheet" href="{{url_for('static', filename='new-styles.css')}}">
+    <link href="https://cdn.jsdelivr.net/npm/hint.css@3.0.0/hint.min.css" rel="stylesheet">
+    <title>POPROX Newsletter</title>
+	{% block header %}
+	{% endblock %}
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="https://poprox.ai/">POPROX</a>
+            <div id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item"><a class="nav-link" href="https://poprox.ai/">Home</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://poprox.ai/about">About</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://poprox.ai/experimenter">Researchers</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://poprox.ai/newsletter">Newsletter Subscribers</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://poprox.ai/people">People</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://poprox.ai/contact">Contact Us</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="row align-items-center">
+            <!-- Image Section -->
+            <div class="row-md-6 text-center">
+                <img src="{{url_for('static', filename='POPROX24-temp00A-01.svg')}}" alt="logo" width="60%">
```

Comment:
I'm _REALLY_ torn between the current approach (Re-hosting the image in both user-web and static-web) vs. hotlinking from the other web-stack. Both have benefits and drawbacks. If no one has strong oppinions one way or another we should probably just stick with what's already in the PR.